api/v1alpha1/agent/openapi.yaml

```diff
@@ -52,6 +52,40 @@ paths:
             application/json:
               schema:
                 $ref: '../openapi.yaml#/components/schemas/Error'
+  /api/v1/devices/{name}/heartbeat:
```

Comment:
All API changes should have a design doc do can you link that here so we can talk about the proposed solution?

Reply:
No... I made it based on ERM-786...
Can you point me to any design doc of this kind and I'll create a new one

Reply:
Gotcha EDM-786 was not marked as `Ready` so @avishayt was not aware it was being worked on.

Reply:
![image](https://github.com/user-attachments/assets/b67e9900-52b5-4b64-ad47-623cf06e773c)

Reply:
> an you point me to any design doc of this kind and I'll create a new one

You could  simplify to meet your needs

[1] https://docs.google.com/document/d/1t0ymgOUwhkA99DTZNmHISesavagwl3G6OSTctKMndQ4/edit?tab=t.0#heading=h.bs36tijs8419